provider/pkg/googleclient/http.go

```diff
@@ -27,6 +27,7 @@ import (
 	"strings"
 	"time"
 
+	tfutils "github.com/GoogleCloudPlatform/magic-modules/mmv1/third_party/terraform/utils"
```

Comment:
This does pull in some TF SDK stuff which seems superfluous. Would prefer not to have to duplicate the logic for retry-able errors though. We can revisit and implement a slimmer version if we want to shrink the dependencies later.